REvernus/Settings/AppSettings.cs

```diff
@@ -11,6 +11,7 @@ public class AppSettings : ITrackingAware<AppSettings>
         public MarginToolSettings MarginToolSettings { get; set; } = new MarginToolSettings();
         public MarketOrdersTabSettings MarketOrdersTabSettings { get; set; } = new MarketOrdersTabSettings();
         public MarketSettings MarketSettings { get; set; } = new MarketSettings();
+        public NotificationSettings NotificationSettings { get; set; } = new NotificationSettings();
```

Comment:
Notifications should not be part of this branch, code relating to notifications should be removed entirely.

Reply:
Removed